lib/scrabble.rb

```diff
@@ -1,18 +1,51 @@
 class Scrabble
 
+  def initialize
+    @array_of_word = []
+    @count = 0
+  end
+
   def score(word)
-    1
+    if word.length > 6
+      @count += 10
+    end
+    @array_of_word =[]
```

Comment:
On line 12 you set the instance variable @array_of_word to an empty array, but it is also set to be an empty array in your initialize. You don't need both, which would be better to keep? Additionally, when naming keep the data type (array) out of your names. You might have named this word or even better letters since it ends up being an array of each letter of the word that gets passed in.

Reply:
I assume that setting it at the top in initialize is best, because thats what I intended for it to always start as and then as I call it later on in my code I would want to fill it.